app/models/submission.rb

```diff
@@ -14,7 +14,8 @@ def latest_version
   end
 
   def as_json(*)
-    super(only: %i[application_state application_risk application_type updated_at created_at]).merge(
+    super(only: %i[application_risk application_type updated_at created_at]).merge(
+      application_state: state,
```

Comment:
Should this not be `state`?

Reply:
Ah, no, because that would be a breaking change to the API, which is out of scope for this ticket. The JSON payloads passed around between the apps still use a field called `application_state`, and will do until we make a version 2 of the API. All we're doing it handling how that field is represented internally within the apps.